x/concentrated-liquidity/types/keys.go

```diff
@@ -22,12 +22,13 @@ const (
 
 // Key prefixes
 var (
-	TickPrefix         = []byte{0x01}
-	PositionPrefix     = []byte{0x02}
-	PoolPrefix         = []byte{0x03}
-	IncentivePrefix    = []byte{0x04}
-	TickNegativePrefix = []byte{0x00}
-	TickPositivePrefix = []byte{0x01}
```

Comment:
I don't think TickPrefix and TickPositivePrefix should have utilized the same bytes. I changed this in this PR.

Reply:
It does not matter but what is important is that the negative prefix is smaller than the positive prefix. Can you please add a comment reflecting this to make sure we don't miss it accidentally in the future?

Reply:
https://github.com/osmosis-labs/osmosis/pull/4655/commits/9111d22b7a98c1441a54a0529d92c412856e5a74